src/Kafka/CreateKafkaInstance/components/TrialKafkaForm.tsx

```diff
@@ -0,0 +1,271 @@
+import {
+  Button,
+  ButtonVariant,
+  Form,
+  FormGroup,
+  Slider,
+  Text,
+  TextInput,
+  TextVariants,
+  ToggleGroup,
+  ToggleGroupItem,
+  Tooltip,
+} from "@patternfly/react-core";
+import { VFC } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { FormGroupWithPopover } from "../../../shared";
+import { CloudProvidersTiles } from "./CloudProviderTiles";
+import { CloudRegionSelect } from "./CloudRegionsSelect";
+import { StandardKafkaFormProps } from "./StandardKafkaForm";
+
+export type TrialKafkaFormProps = Omit<
+  StandardKafkaFormProps,
+  | "streamingUnits"
+  | "remainingStreamingUnits"
+  | "allowedStreamingUnits"
+  | "sizeValidation"
+> & {
+  onClickLearnMoreAboutRegions: () => void;
+};
+
+export const TrialKafkaForm: VFC<TrialKafkaFormProps> = ({
+  FORM_ID,
+  isNameTaken,
+  isNameInvalid,
+  nameValidation,
+  name,
+  disableControls,
+  providerValidation,
+  availableProviders,
+  provider,
+  regionValidation,
+  regions,
+  region,
+  azValidation,
+  azOptions,
+  az,
+  disableAZTooltip,
+  isDisabledSize,
+  size,
+  setSize,
+  setRegion,
+  setAZ,
+  setProvider,
+  setName,
+  onSubmit,
+  instanceAvailability,
+  onClickLearnMoreAboutRegions,
+}) => {
+  const { t } = useTranslation("create-kafka-instance-exp");
+
+  const allRegionsUnavailable =
+    regions?.every(({ isDisabled }) => isDisabled === true) ||
+    (regions && regions?.length <= 0);
+
+  const someRegionsUnavailable =
+    regions && regions?.some(({ isDisabled }) => isDisabled === true);
+
+  const HelperText = () => {
+    const ErrorMessage = () => {
+      switch (true) {
+        case allRegionsUnavailable &&
+          regionValidation === "error" &&
+          !disableControls:
+          return (
+            <div className="pf-c-form__helper-text pf-m-error">
+              {t("trial_all_region_unavailable_helper_text")}
+            </div>
+          );
+        case someRegionsUnavailable &&
+          regionValidation === "warning" &&
+          !disableControls:
+          return (
+            <div className="pf-c-form__helper-text pf-m-warning">
+              {t("trial_some_region_unavailable_helper_text")}
+            </div>
```

Comment:
The  FieldCloudRegion component was updated to use the HelperText component instead. Do the same updates need to be made here?